ditados.json

```diff
@@ -3,6 +3,10 @@
       {
          "content":"É desde pequenino que se torce o pepino",
          "meaning":"Os bons costumes e educação que se dá as crianças deve ser feito desde o mais cedo possível."
+      },
+      {
+         "content":"Quem não arrisca, não petisca.",
+         "meaning":"As vezes correr um risco pode ter benefícios."
```

Comment:
Missing an accent here on the capital 'A', i.e, À

Reply:
Ah, boa. Fixed :)

Reply:
Sorry, wrong accent 😅

Reply:
Hah, sorry. Fixed it, this is what happens when I'm careless.